### DISCUSSION TITLE
pion webrtc support both ice-tcp and ice-udp

### DISCUSSION THREAD

**Discussion by avStudy** [pion/webrtc#2122] [General] [answered]
If the pion webrtc supports both ice-udp and ice-tcp, the default ice-udp priority?

**Comment by Sean-Der** [comment, marked as answer]
Hi @avStudy 

Yes Pion supports both UDP and TCP for ICE. We do default to UDP if we are the controlling agent.

**Comment by avStudy** [comment]
Hi @Sean-Der 
Thanks a million! Achieve results as good as you say.
